.github/workflows/magma-build-3rd-party.yml

```diff
@@ -112,9 +113,12 @@ jobs:
         if: steps.jfrog-setup.conclusion == 'success' && github.event_name == 'workflow_dispatch'
         working-directory: third_party/build
         run: |
-          name=$(ls *${{ matrix.dependency }}*.deb) && architecture=${${name%%.deb}##*_}
+          file=$(ls *${{ matrix.dependency }}*.deb)
+          name=${file%%.deb}
+          architecture=${name##*_}
+          echo Architecture is ${architecture}
```

Comment:
Apparently nested string operations seem to be a problem, such that I rather spelled this out, step by step.